include/secp256k1.h

```diff
@@ -134,7 +134,7 @@ typedef int (*secp256k1_nonce_function)(
 #endif
 
 /* Symbol visibility. See libtool manual, section "Windows DLLs". */
```

Comment:
Can you update this comment to say that we slightly deviate from the suggestion in the libtool manual because the mingw linker apparently respects `_declspec (dllexport)` but not `_attribute__ ((visibility ("default")))`?

Reply:
https://www.gnu.org/software/libtool/manual/html_node/Windows-DLLs.html

Reply:
Updated to include our reference as opposed to what we _don't_ use. This ok?